tests/wpt/mozilla/tests/mozilla/mozbrowser/mozbrowserlocationchange_event.html

```diff
@@ -55,5 +55,16 @@
 
 });
 
+async_test(function(t) {
+  var iframe = document.createElement("iframe");
+  iframe.mozbrowser = "true";
+  iframe.src = "mozbrowserlocationchange_event_iframe.html";
+  iframe.addEventListener("mozbrowserlocationchange", function(e) {
```

Comment:
step_func also catches any exceptions that might be thrown. It's good practice to use it in WPT tests whenever possible.